plugin/pulp/plugin/__init__.py

```diff
@@ -1,4 +1,5 @@
 from pulp.app.models import ProgressBar, ProgressSpinner, Repository, RepositoryContent
+from pulp.exceptions import PulpException, MissingResource
```

Comment:
I did not add the MissingResource import here. I just added the PulpException and a newline.